### ISSUE TITLE
project alive?

### ISSUE THREAD

**Issue by bboerni2** [VoidArkMods/QoLPlus#119]
Is this Project still alive?

**Comment by MasterBLaha** [comment]
No Discord, No Comments on CurseForge, No Idea...

**Comment by hagen843** [comment]
I asked myself too if it is still being worked on... I would be very sad if not...

**Comment by bboerni2** [comment]
yes would be nice if one of the devs could give some informations

**Comment by NobleKangaroo** [comment]
Seems like every other day there's more things that stop working - transferring to Tek Dedicated Storage with the Omni Tool, metal storage box snapping, now QOL+ refrigerators stopped snapping.. Would try to contribute but the barrier to entry with Unreal Engine projects and usset files is much greater than just forking a repo, editing code, and sending a pull request. Hoping @VoidArkMods isn't abandoning this.

**Comment by zerafox** [comment]
And here I am hoping there's been at least a reply or something to one of these issues as we are also getting a bit concerned that the project has been abandoned.
Some kinds of heads up would be nice at the very least.

**Comment by GUGU183412** [comment]
no！

**Comment by zerafox** [comment]
It's been since Dec 3rd since the last update of this mod and not a word from the Team for quite some time.
I'm really starting to think that this has in fact been abandoned despite Curse Forge showing that the creator has been online 2 days ago.

If someone knows something I don't, please share.
Otherwise, I'm looking for alternatives already because this lack of communication isn't sitting well with me

**Comment by VoidArkMods** [comment]
Hello!
The mod has not been abandonned, I just slowed down my game on TheIsland. Initially, we made the mod to support our gameplay, re-adding little by little the features we preferred from S+. I'm not payed to do that, and don't want to be, this is a pure hobby, during my free time, when I get some.
I'm aware of the snapping feature, every QoL+ structure has been directly imported from S+, the way they snap included. A lot has changed in ASA wrt snapping, causing the air conditionneer to not snap anymore to windows for example (as walls and windows changed drastically). It's annoying for sure, but this is not game-breaking. As such, fixing the snapping is a bit low on the priority list. It's likely that some features will make it to the mod before the snapping gets fixed (for example, QoL+ dedicated storage + intake, tek-level crafting structures, transmitter with dino scan, etc).
